### PR TITLE
[KUBOS-326] Remote Update Doc

### PR DESCRIPTION
**Pull request by CatFreed**
Created a new document to cover KubOS Linux upgrade package creation, installation, and rollback.

### CODE REVIEW

File: docs/kubos-linux-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+The SD card should have been formatted with the correct partitions. If not, refer to the 'Installation Process->Partition the SD Card' section
+of the [KubOS Linux on the ISIS iOBC](docs/kubos-linux-on-iobc.md) doc.
+
+The host computer should be connected to the board and the iOBC should be on and running KubOS Linux. 
+
+A Kubos SDK VM should be installed on your host computer and atleast one shared folder should be set up.  Installation instructions can be found [here](docs/sdk-installing.md).
+
+### Installation
+
+Acquire an upgrade package. For now, this will likely be sent via email from a Kubos engineer.  Once an official distribution process has been created
```
Comment (kyleparrott) [inline docs/kubos-linux-upgrade.md]: I think we should investigate alternative approaches for generating all these files for flashing this package. This is fine for a first iteration but we can come up with a better UX than creating a fake project for longer term 😃

File: docs/kubos-linux-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
+# Upgrading KubOS Linux
+
+[TOC]
+
+# Upgrading KubOS Linux {#kubos-linux-upgrade}
```
Comment (CatFreed) [inline docs/kubos-linux-upgrade.md]: It's used by Doxygen to create the table of contents when we generate our official doc. Github's markdown doesn't know how to process it.

File: docs/kubos-linux-upgrade.md (hunk 1 above)
Comment (CatFreed) [inline docs/kubos-linux-upgrade.md]: :+1:

File: docs/kubos-linux-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+Within each file will be a new version of the kernel image and root filesystem.
+
+To upgrade a board currently running KubOS Linux, an upgrade package will be loaded into the upgrade partition of the board. For now, this can be done through the Kubos SDK or by 
+manually copying the package into the upgrade partition.
+
+Once the board is rebooted, U-Boot will take the package and then install each component into the appropriate partition (kernel/rootfs). If installation completes successfully, then
+the board will be rebooted and then boot into the new version of KubOS Linux.
+
+Note: User files should not be impacted by upgrade installation as long as they remain under the /home directory. This directory maps to the userspace partition.
```
Comment (plauche) [inline docs/kubos-linux-upgrade.md]: We might want to add a note about Kubos libraries changing (which user applications may depend on).

Comment (CatFreed) [inline docs/kubos-linux-upgrade.md]: Good point, I'll add something. Once we get stable, however, we'll (probably) want to make sure code changes are backwards compatible.

**Pull request merged**